src/Sylius/Bundle/AddressingBundle/Form/Type/CountryChoiceType.php

```diff
@@ -25,14 +25,14 @@ class CountryChoiceType extends AbstractType
     /**
      * @var RepositoryInterface
      */
-    protected $countryRepository;
+    protected $repository;
```

Comment:
It is a country choice type and there is no other repository so I guess we can leave it like that, wdyt?